tools/checked-c-convert/CheckedCConvert.cpp

```diff
@@ -77,29 +79,38 @@ const Type *getNextTy(const Type *Ty) {
 }
 
 // Test to see if we can rewrite a given SourceRange. 
+// Note that R.getRangeSize will return -1 if SR is within
+// a macro as well. 
```

Comment:
Please clarify that the clang rewriter infrastructure won't let us rewrite macros at this time.   This is because R.getRangeSize returns -1 if SR is within a macro as well.